packages/website/src/components/search/search.css

```diff
@@ -1,29 +1,81 @@
-.ma-search {
-  column-gap: var(--basis-space-inline-md);
-  container-name: ma-search;
-  container-type: inline-size;
+/* stylelint-disable -- Overwriting DocSearch, we don't have control over the markup. */
+.DocSearch {
+  --ma-docsearch-button-border-color: var(--utrecht-button-secondary-action-border-color);
```

Comment:
Is al die indirecte code worth it? Lijkt het je onhandig om in de code gelijk de secondary button code variables in te stellen?

Het lijkt me dat je in een ideale wereld niet ooit een officiele docsearch-button component hebt, maar dat die gewoon opgebouwd is met de juiste button component. Dus je werkt ook niet al stapsgewijs toe naar de ideale situatie.